### PR TITLE
refactor: rename `suggestion-input` to `auto-suggest-input`

### PR DESCRIPTION
**Pull request by eliflores**
related to https://github.com/digitalservicebund/a2j-rechtsantragstelle-strapi/pull/275

### CODE REVIEW

**Comment by aaschlote** [comment]
Missing the story file `SuggestionInput.stories.tsx` to rename  and the function fillMultipleSuggestionInputPage

File: stories/SuggestionInput.stories.tsx [hunk 1]
```diff
@@ -31,18 +31,18 @@ const defaulType: SuggestionInputProps = {
 
 const meta = {
   title: "Component/SuggestionInput",
```
Comment (aaschlote) [inline stories/SuggestionInput.stories.tsx:33]: here as well

File: stories/SuggestionInput.stories.tsx [hunk 2]
```diff
@@ -19,7 +19,7 @@ function getDataListValues<T extends { dataList?: any }>(
   );
 }
 
-const defaulType: SuggestionInputProps = {
+const defaulType: AutoSuggestInputProps = {
   name: "suggestionInput",
```
Comment (aaschlote) [inline stories/SuggestionInput.stories.tsx:23]: we need to rename here too

**Review by aaschlote — CHANGES_REQUESTED** [review]

File: stories/SuggestionInput.stories.tsx (hunk 1 above)
Comment (eliflores) [reply stories/SuggestionInput.stories.tsx]: Good catch ✨ 
Done in [fbf9e82](https://github.com/digitalservicebund/a2j-rechtsantragstelle/pull/888/commits/fbf9e827bfcbe014ccbba05aac8a84c2579006ee)

File: stories/SuggestionInput.stories.tsx (hunk 2 above)
Comment (eliflores) [reply stories/SuggestionInput.stories.tsx]: 👍 Done in [fbf9e82](https://github.com/digitalservicebund/a2j-rechtsantragstelle/pull/888/commits/fbf9e827bfcbe014ccbba05aac8a84c2579006ee)

**Review by aaschlote — APPROVED** [review]

**Pull request merged**